Kickstarter-iOS/Tests/Views/ProjectPamphletContentViewControllerTests.swift

```diff
@@ -32,7 +32,7 @@ internal final class ProjectPamphletContentViewControllerTests: TestCase {
 
   override func tearDown() {
     AppEnvironment.popEnvironment()
-    UIView.setAnimationsEnabled(true)
+    UIView.setAnimationsEnabled(false)
```

Comment:
mostly just in case we decide we wanna use animations in a test for some reason (¯\\_(ツ)_/¯), and the default is `true` so we preserved it.

we could totally do `UIView.setAnimationsEnabled(false)` in our base test case so that we can remove all this, and then if a test wants to do something with animation it would opt into it.